lol.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <h1>LOL</h1>
+    <p>
+    리그오브레전드<br/>
+    리그오브레전드는 라이엇에서 제작했으며 2011년12월4일에 출시한 AOS장르의 게임이다.<br/>
+    끝
+
+    </p>
```

Comment:
끝 이라는 말은 안해도 되지 않을까요?

```suggestion
    <p>
    리그오브레전드<br/>
    리그오브레전드는 라이엇에서 제작했으며 2011년12월4일에 출시한 AOS장르의 게임이다.<br/>
    </p>
```